x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/json_diff/json_diff.test.tsx

```diff
@@ -0,0 +1,290 @@
+      />
+    );
+    expect(screen.queryAllByText(matchInOrder(['actions']))).toHaveLength(0);
+
+    /* Case: rule update has an action and an exception list item */
+    rerender(
+      <RuleDiffTab
+        oldRule={{ ...oldRule }}
+        newRule={{
+          ...newRule,
+          actions: [{ ...testAction, id: 'my-other-action' }],
+          exceptions_list: [testExceptionListItem],
+        }}
+      />
+    );
+    expect(screen.queryAllByText(matchInOrder(['actions']))).toHaveLength(0);
+  });
+
+  it('Technical properties should not be included in preview', () => {
+    const oldRule: RuleResponse = {
+      ...savedRuleMock,
+      version: 1,
+      revision: 100,
+      created_at: '12/31/2023T23:59:999z',
+      created_by: 'mockUserOne',
+      updated_at: '01/01/2024T00:00:000z',
+      updated_by: 'mockUserTwo',
+    };
+
+    const newRule: RuleResponse = {
+      ...savedRuleMock,
+      version: 2,
+      revision: 1,
+      created_at: '12/31/2023T23:59:999z',
+      created_by: 'mockUserOne',
+      updated_at: '02/02/2024T00:00:000z',
+      updated_by: 'mockUserThree',
+    };
+
+    render(<RuleDiffTab oldRule={oldRule} newRule={newRule} />);
+    ['revision', 'created_at', 'created_by', 'updated_at', 'updated_by'].forEach((property) => {
+      expect(screen.queryAllByText(matchInOrder([property]))).toHaveLength(0);
+    });
+  });
+
+  it('Properties with semantically equal values should not be shown as modified', () => {
+    const oldRule: RuleResponse = {
+      ...savedRuleMock,
+      version: 1,
+    };
+
+    const newRule: RuleResponse = {
+      ...savedRuleMock,
+      version: 2,
+    };
+
+    /* DURATION */
+    /* Semantically equal durations should not be shown as modified */
+    const { rerender } = render(
+      <RuleDiffTab
+        oldRule={{ ...oldRule, from: 'now-1h' }}
+        newRule={{ ...newRule, from: 'now-60m' }}
+      />
+    );
+    expect(screen.queryAllByText(matchInOrder(['-', 'from', '+', 'from']))).toHaveLength(0);
+
+    rerender(
+      <RuleDiffTab
+        oldRule={{ ...oldRule, from: 'now-1h' }}
+        newRule={{ ...newRule, from: 'now-3600s' }}
+      />
+    );
+    expect(screen.queryAllByText(matchInOrder(['-', 'from', '+', 'from']))).toHaveLength(0);
+
+    rerender(
+      <RuleDiffTab
+        oldRule={{ ...oldRule, from: 'now-7200s' }}
+        newRule={{ ...newRule, from: 'now-2h' }}
+      />
+    );
+    expect(screen.queryAllByText(matchInOrder(['-', 'from', '+', 'from']))).toHaveLength(0);
+
+    /* Semantically different durations should generate diff */
+    rerender(
+      <RuleDiffTab
+        oldRule={{ ...oldRule, from: 'now-7260s' }}
+        newRule={{ ...newRule, from: 'now-2h' }}
+      />
+    );
+    expect(screen.queryAllByText(matchInOrder(['-', 'from', '+', 'from']))).toHaveLength(1);
+    expect(
+      screen.queryAllByText(matchInOrder(['-', 'from', 'now-7260s', '+', 'from', 'now-7200s']))
+    ).toHaveLength(1);
+
+    /* NOTE */
```

Comment:
nit: 
```
/* NOTE - Investigation guide */
```

Otherwise looks like an empty note 😆

Reply:
Addressed in a follow-up commit